bentoml/handlers/json_handler.py

```diff
@@ -45,8 +50,42 @@ def handle_request(self, request, func):
         json_output = api_func_result_to_json(result)
         return Response(response=json_output, status=200, mimetype="application/json")
 
-    def handle_batch_request(self, requests, func):
-        raise NotImplementedError
+    def handle_batch_request(
+        self, requests: Iterable[SimpleRequest], func
+    ) -> Iterable[SimpleResponse]:
+        bad_resp = SimpleResponse(400, None, "Bad Input")
+        instances_list = [None] * len(requests)
+        responses = [bad_resp] * len(requests)
+
+        for i, request in enumerate(requests):
+            try:
+                raw_str = request.data
+                parsed_json = json.loads(raw_str)
+                instances_list[i] = parsed_json
+            except (json.JSONDecodeError, UnicodeDecodeError):
+                responses[i] = SimpleResponse(400, None, "not a valid json input")
+            except Exception:  # pylint: disable=broad-except
```

Comment:
any specific reason that we need this? I think the API sever side does a broad except and will handle unexpected errors

Reply:
To catch the error so that other normal requests could be properly returned.

Reply:
👍 got it